### PR TITLE
images: Install ws container into all Fedora images

### PR DESCRIPTION
**Pull request by martinpitt**
Not having it on Fedora 41 makes it impossible to upgrade the container to fedora:41 and properly test it. We may also want to test the container on more Fedora versions. The container is tiny (57 MB compressed, 147 MB uncompressed), so this doesn't hurt much.

 * [x] image-refresh fedora-41

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh fedora-41 done: https://github.com/cockpit-project/bots/commits/image-refresh-fedora-41-20241129-141343

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-fedora-41-54933bf7-20241129-135554/log.html

File: images/scripts/fedora.setup [hunk 1]
```diff
@@ -213,11 +213,9 @@ dnf install -y $selinux_pkgs
 # for c-podman tests
 /var/lib/testvm/podman-images.setup
 
-# pre-install cockpit/ws container for /ws-container scenario; we only need that on the image which the container is based on
+# pre-install cockpit/ws container for /ws-container scenario
 CF=https://raw.githubusercontent.com/cockpit-project/cockpit/main/containers/ws/Dockerfile
-if curl --silent --show-error --fail "$CF" | grep -q "^FROM.*fedora:$VERSION_ID"; then
-    podman pull quay.io/cockpit/ws:latest
-fi
+podman pull quay.io/cockpit/ws:latest
```
Comment (jelly) [inline images/scripts/fedora.setup:218]: Does this interfere with coreos? This also has FEDORA has ID.

**Review by jelly — COMMENTED** [review]
Small question

File: images/scripts/fedora.setup (hunk 1 above)
Comment (martinpitt) [reply images/scripts/fedora.setup:218]: No, it doesn't interfere. fedora-coreos has its own .setup, only fedora-{39,40,41} use this script.

**Review by jelly — APPROVED** [review]
Thanks!

**Pull request merged**